package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-output",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
can undo this change. will bump the version for the release using the `release:patch` script